cmd/vcfToFa/vcfToFa.go

```diff
@@ -53,16 +97,18 @@ func vcfToMultiFa(inVcfFilename string, inFaFilename string, outFaFilename strin
 		if !(vcf.IsBiallelic(v) && vcf.IsSubstitution(v)) {
```

Comment:
Does this mean that if any vcf record contains more than 2 alleles, e.g. ref=A, alt=AT, the whole program would log fatal? Is there a situation where it would be easier for the user if the program had given a warning instead, e.g. if they  have more than 2 alleles somewhere in their vcf, but still wanted to run this program without specifically removing those records?

Reply:
Yes. one could simply remove the offending records with vcfFilter and run this program again. I was initially worried about continuing because it could produce some silent errors, the warning might be a viable compromise.

Reply:
I am always nervous to have the user tell us to do something and us to just silently not do it.  The warning is a good idea, but I am worried that the user may not see it with cluster jobs.  I have updated the error message to let the user know they can use vcfFilter to update their vcf file (if they are ok with only biallelic substitutions.